experimental/addons/redis-exporter/README.md

```diff
@@ -0,0 +1,174 @@
+            metadata:
+              name: <redis-secret-name>
+            data:
+              username: <username-bash64> # default: ""
+              password: <password-bash64> # default: password in base64
+    - name: redis
+      type: helm
+      properties:
+        repoType: "helm"
+        url: "https://charts.bitnami.com/bitnami"
+        chart: "redis"
+        version: "17.7.3"
+        values:
+          global:
+            redis:
+              password: <password>
+          master:
+            persistence:
+              size: 16Gi
+          replica:
+            persistence:
+              size: 16Gi
+      traits:
+        - type: redis-exporter
+          properties:
+          address: <host>:<port>
+          secretName: <redis-secret-name>
+          disableAnnotation: false
+          name: redis-server-exporter
+```
+
+## Exposed Metrics
+
+### Access by `port-forward`
+
+Base use case above, you can get the metrics exposed from exporter by `vela port-forward`:
+
+```shell
+vela port-forward <app-name> 9121:9121
+```
+
+Then select the `redis-server-exporter` service:
+
+```shell
+vela port-forward redis 9121:9121
+? There are 4 services match your filter conditions. Please choose one:
+Cluster | Component | Service  [Use arrows to move, type to filter]
+  local | redis | redis-headless
+  local | redis | redis-master
+  local | redis | redis-replicas
+> local | redis-exporter | redis-exporter
+```
+
+After that, you can access the metrics by `http://localhost:9121/metrics`.
+
+```
+...
+# HELP redis_active_defrag_running active_defrag_running metric
+# TYPE redis_active_defrag_running gauge
+redis_active_defrag_running 0
+# HELP redis_allocator_active_bytes allocator_active_bytes metric
+# TYPE redis_allocator_active_bytes gauge
+redis_allocator_active_bytes 1.728512e+06
+# HELP redis_allocator_allocated_bytes allocator_allocated_bytes metric
+# TYPE redis_allocator_allocated_bytes gauge
+redis_allocator_allocated_bytes 1.340704e+06
+# HELP redis_allocator_frag_bytes allocator_frag_bytes metric
+# TYPE redis_allocator_frag_bytes gauge
+redis_allocator_frag_bytes 387808
+# HELP redis_allocator_frag_ratio allocator_frag_ratio metric
+# TYPE redis_allocator_frag_ratio gauge
+redis_allocator_frag_ratio 1.29
+...
+```
+
+### Grafana Dashboard
+
+You can choose to install Grafana in the Kubernetes cluster manually, or use Grafana in a simple way by installing the [Grafana addon](https://github.com/kubevela/catalog/tree/master/addons/grafana) of KubeVela. And then you can use dashboard to visualize the metrics.
+
+Example Grafana screenshots:
+
+![](https://cloud.githubusercontent.com/assets/1222339/19412041/dee6d7bc-92da-11e6-84f8-610c025d6182.png)
+
+Grafana dashboard is available on [grafana.com](https://grafana.com/grafana/dashboards/763-redis-dashboard-for-prometheus-redis-exporter-1-x/) and/or [github.com](https://github.com/oliver006/redis_exporter/blob/master/contrib/grafana_prometheus_redis_dashboard.json).
```

Comment:
great, can we just make the grafana dashboard as part of this addon? Check the magic here: https://kubevela.net/zh/docs/platform-engineers/operations/o11y/dashboard

Reply:
How about now? Instead of taking it as a part of the addon, I provide an example of importing Dashboard through workflow in README.